airflow/datasets/__init__.py

```diff
@@ -42,7 +42,12 @@ def _get_uri_normalizer(scheme: str) -> Callable[[SplitResult], SplitResult] | N
     return ProvidersManager().dataset_uri_handlers.get(scheme)
 
 
-def _sanitize_uri(uri: str) -> str:
+def sanitize_uri(uri: str) -> str:
+    """Sanitize a dataset URI.
```

Comment:
Should we add `:meta: private`? or are we making this function public?

Reply:
I’ll add the meta.